source/gui/configProfiles.py

```diff
@@ -482,6 +484,8 @@ def __init__(self, parent):
 		self.Sizer = mainSizer
 		self.profileName.SetFocus()
 		self.CentreOnScreen()
+		# Note: we don't call guiHelper.enableDarkMode() here because wx.RadioBox doesn't support
```

Comment:
Is there a ticket agains wx about this?
Could you add the ref here?

Reply:
Done!